.werft/platform-trigger-artificial-job.sh

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env bash
+#
+# The easiest way to run this script is through Werft so you don't have to worry
+# about installing the appropraite service account etc. locally.
+#
+#   werft job run github -j .werft/platform-trigger-artificial-job.yaml -s .werft/platform-trigger-artificial-job.sh
+#
+
+sleep 1
+
+set -Eeuo pipefail
+
+werft log phase "Trigger new job with-preview" "Trigger new job with preview environment"
+sudo chown -R gitpod:gitpod /workspace
+git config --global user.name roboquat
+git config --global user.email roboquat@gitpod.io
```

Comment:
If we're not pushing to origin I wonder if we even need these three lines at all ☺️

Reply:
Unfortunately it is still needed :(

Reply:
![image](https://user-images.githubusercontent.com/24193764/188666928-603b3412-7fa7-4f4f-a2cf-d70290e20700.png)